lib/SILGen/SILGenLValue.cpp

```diff
@@ -1588,7 +1589,7 @@ namespace {
       assert(getAccessorDecl()->isGetter());
 
       SILDeclRef getter = Accessor;
-      bool didHop = false;
+      ExecutorBreadcrumb didHop;
```

Comment:
`didHop` made sense as a boolean, but doesn't seem clear in the current use. I see that below, in `SILGenFunction::emitHopToTargetActor`, you're just calling it `breadcrumb`. Can we change the name to more accurately describe the use?